api/applicationoffers/client_test.go

```diff
@@ -668,7 +669,7 @@ func (s *crossmodelMockSuite) TestDestroyOffers(c *gc.C) {
 			return nil
 		})
 	client := applicationoffers.NewClient(apiCaller)
-	err := client.DestroyOffers("me/prod.app")
+	err := client.DestroyOffers(true, "me/prod.app")
```

Comment:
You'd need to check that for v1, the force parameter is not needed...

Reply:
I added a version check and a new test.